lib/phlex/test_helpers.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Phlex
+	module TestHelpers
+		def render_view(view, &block)
```

Comment:
I wonder if we should use `render` here instead. What do you think? I’m not sure if that’s defined by anything else.

```suggestion
		def render(view, &block)
```

Reply:
RSpec defines `render` method for view tests: https://github.com/rspec/rspec-rails/blob/main/lib/rspec/rails/example/view_example_group.rb#L66

Reply:
Hmm, I think we might even be able to use their definitions of render.

Reply:
This will make the test helper depend directly on rails. I changed the
```ruby
rendered = controller.view_context.render(view, &block)
Nokogiri::HTML.fragment(rendered)
```

To:
```ruby
view_context = controller&.view_context # controller can be nil if ActionView::TestCase::TestController is not defined
rendered = view.call(view_context: view_context, &block)
Nokogiri::HTML.fragment(rendered)
```

I tested the new code with:
- Rails + RSpec
- Rails + Minitest 
- Ruby Gem (No Rails) + RSpec
- Ruby Gem (No Rails) + Minitest

And it's working fine. I'm finishing the docs to push the changes. If we're not happy with `render_view`, we can change to another name, eg. `render_inline`.
What do you think?

Reply:
This is nice.